python/lsst/sims/utils/CompoundCoordinateTransformations.py

```diff
@@ -154,23 +131,8 @@ def _raDecFromAltAz(altRad, azRad, obs):
     Note: This method is only accurate to within 0.01 arcsec near azimuth = 0 or pi
     """
 
-    altIsArray = False
-    azIsArray = False
-
-    if isinstance(altRad, np.ndarray):
-        altIsArray = True
-
-    if isinstance(azRad, np.ndarray):
-        azIsArray = True
-
-    if altIsArray and not azIsArray:
-        raise RuntimeError('passed a numpy array of alt to raDecFromAltAz, but only one az')
-
-    if azIsArray and not altIsArray:
-        raise RuntimeError('passed a numpy array of az to raDecFromAltAz, but only one alt')
-
-    if azIsArray and altIsArray and len(altRad)!=len(azRad):
-        raise RuntimeError('in raDecFromAltAz, length of alt numpy array does not match length of az numpy array')
+    are_arrays = _validate_inputs([altRad, azRad], ['altRad', 'azRad'],
+                                 "raDecFromAltAz")
```

Comment:
Maybe it's just that you're using are_arrays here for the same "other validating" purposes as you mentioned in some other cases. I was just thinking that are_arrays would return the same value as it did for the ra/dec, so it seemed unnecessary to call it again on the alt/az values.